wordpress/Caddyfile

```diff
@@ -1,2 +1,10 @@
 localhost:8080
-fastcgi / unix:/var/run/php5-fpm.sock php
+root ./html/
```

Comment:
We can probably omit this root directive since it's going to vary depending on each user's setup.